modules/channel-web/src/backend/api.ts

```diff
@@ -94,6 +94,12 @@ export default async (bp: typeof sdk, db: Database) => {
       await bp.users.getOrCreateUser('web', userId) // Create the user if it doesn't exist
 
       const payload = req.body || {}
+      const { timezone } = payload
+
+      if (timezone) {
+        await bp.users.updateAttributes('web', userId, { timezone })
+      }
```

Comment:
Nothing prevents the user from crafting a string other than a timezone

Reply:
@allardy If I add the following:
```js
if (timezone && !/\D+/.test(timezone)) {
   await bp.users.updateAttributes('web', userId, { timezone })
}
```
It will resolve this issue?

Reply:
@allardy If I add the following:

```js
if (timezone && !/\D+/.test(timezone)) {
   await bp.users.updateAttributes('web', userId, { timezone })
}
```

It will resolve this issue?

Reply:
@allardy If I add something like this:
```js
if (timezone && _.isNumber(timezone)) {
   await bp.users.updateAttributes('web', userId, { timezone })
}
```
Will it resolve this issue?